src/test/java/complete_task/completeTaskUseCaseTest.java

```diff
@@ -32,9 +31,6 @@ public static void setup(){
         schedule = new PrebuiltScheduleFactory().create();
         activeUser = new CommonUserFactory().create
                 ("username", "email@email.com", "password");
-        //readTextbook = new CommonTaskFactory().create("Read Textbook", "Read chapter 1 of Clean Architecture");
-        //attendClass = new CommonTaskFactory().create("Attend Lecture", "Tuesday 6-8pm at Bahen Centre");
```

Comment:
I believe this is the code causing the bug so good thing you commented it out. Good use of the PrebuiltScheduleFactory() and the CommonUserFactory, to incorporate the elements of the schedule and activeUser.